docs/technical_reference/unit_models/Biploar_electrodialysis_0D.rst

```diff
@@ -0,0 +1,340 @@
+Bipolar Electrodialysis (0D)
+============================
+
+Introduction
+------------
+
+Bipolar electrodialysis, an electrochemical separation technology, has primarily been used to generate acids and bases
+from waste salts produced in water purification. By providing in-situ access to valuable raw materials bipolar membranes can
+significantly reduce the total cost of the operation.  This cell stack is shown in Figure 1 with **basic** and **acidic** channels, that produce base and acid
+respectively. More overview of the bipolar
+electrodialysis technology can be found in the *References*.
+
+.. figure:: ../../_static/unit_models/BPEDdiagram.png
+    :width: 600
+    :align: center
+
+    Figure 1. Schematic representation of a bipolar electrodialysis unit
+
+
+One bipolar membrane along with the **acidic** and **basic** channels can thus be treated as a modelling unit that can
+multiply to larger-scale systems. The presented bipolar electrodialysis model establishes mathematical descriptions of
+ion and water transport across the membrane along with water splitting. Modelled transfer mechanisms include
+electrical migration, diffusion of ions, osmosis, electroosmosis, and water splitting. The following are the key
+assumptions made:
```

Comment:
@adam-a-a This is a good question. I actually faced the other problem. It was challenging finding the appropriate steady state equations for bipolar membrane (there are many equations/features  for dynamic bipolar modelling but that's a can of worms for another day). 
Even now there are some features (like the boundary layer details) that have not been added to the unit model because I can't find appropriate equations.